xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_ru.properties

```diff
@@ -3633,7 +3633,7 @@ extensions.info.category.progress=\u041F\u0440\u043E\u0433\u0440\u0435\u0441\u04
 extensions.info.id=ID
 extensions.info.type=\u0422\u0438\u043F
 extensions.info.license={0,choice,0#\u041B\u0438\u0446\u0435\u043D\u0437\u0438\u044F \u043D\u0435 \u0438\u0437\u0432\u0435\u0441\u0442\u043D\u0430|1#\u041B\u0438\u0446\u0435\u043D\u0437\u0438\u044F|1<\u041B\u0438\u0446\u0435\u043D\u0437\u0438\u0438(\u0439)}
-extensions.info.features={0,\u0432\u044B\u0431\u043E\u0440,0#\u041D\u0435\u0442 \u0444\u0443\u043D\u043A\u0446\u0438\u0439|1#\u0424\u0443\u043D\u043A\u0446\u0438\u044F|1<\u0424\u0443\u043D\u043A\u0446\u0438\u0438(\u0439)}
+extensions.info.features={0,choice,0#\u041D\u0435\u0442 \u0444\u0443\u043D\u043A\u0446\u0438\u0439|1#\u0424\u0443\u043D\u043A\u0446\u0438\u044F|1<\u0424\u0443\u043D\u043A\u0446\u0438\u0438(\u0439)}
```

Comment:
This "choice" word is not actually English content to translate, but is part of the syntax of the message (it indicates that following is a choice of content which depends on the value of the parameter).